app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,19 @@
+class GroupsController < ApplicationController
+    include GroupsHelper
+
+    def show
+        @group = Group.find(params[:id])
+        @projects = Project.all
+    end
+    
+    def update
+        @group = Group.find(params[:id])
+        @projects = Project.all
+        @group[:first_choice] = params[:first_choice]
+        @group[:second_choice] = params[:second_choice]
+        @group[:third_choice] = params[:third_choice]
```

Comment:
Is there any way we can simplify this logic so it can take an arbitrary set of choices?